cmd/estuary-shuttle/main.go

```diff
@@ -1876,7 +1931,7 @@ func writeAllGoroutineStacks(w io.Writer) error {
 
 func (s *Shuttle) handleRestartAllTransfers(e echo.Context) error {
 	ctx := e.Request().Context()
-	transfers, err := s.Filc.TransfersInProgress(ctx)
+	transfers, err := s.Filc.V110TransfersInProgress(ctx)
```

Comment:
what is the difference between both versions?

Reply:
`V110TransfersInProgress` returns transfers in progress for legacy markets deals.
`TransfersInProgress` returns _all_ transfers in progress (both legacy markets deals and Boost deals).
Note that in Boost deals the Storage Provider downloads from the client (estuary), so the restart happens on the Storage Provider side. For legacy deals the restart happens on the client (estuary) side.